airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/CdcWalLogsPostgresSourceDatatypeTest.java

```diff
@@ -196,25 +196,6 @@ protected void addTimeWithTimeZoneTest() {
     }
   }
 
-  @Override
-  protected void addTimestampWithInfinityValuesTest() {
```

Comment:
removed this because 1) it's testing the wrong thing and 2) we have a lot more infinity tests in the abstract class above